package.json

```diff
@@ -28,10 +28,11 @@
     "node": ">= 6.0"
   },
   "dependencies": {
+    "@airbnb/node-memwatch": "^1.0.2",
     "es6-error": "^4.0.2",
     "left-pad": "^1.1.3",
-    "memwatch-next": "^0.3.0",
     "minimist": "^1.2.0",
+    "nan": "^2.10.0",
```

Comment:
Quick question: Why is `nan` necessary now?

Reply:
it's introduced because of a dependency problem deep in memwatch. Somehow if you don't specify a newer nan, an older version is installed and therefore memwatch cannot be installed because of https://github.com/nodejs/nan/issues/504.

Having said that, I think I should declare it as a peer dependency rather than a direct one, agree?

Reply:
Good question... But since users don‘t know about this nasty little trick (and they shouldn‘t have to care) it might be best to keep it as a real dependency.

We should just document it somewhere.

Reply:
In fact, I really don't like this trick. Am investigating other means to get rid of it.